README.md

```diff
@@ -40,36 +39,30 @@ julia> using Pkg
 julia> Pkg.add(ClimaLand)
 ```
 
-Which is equivalent to doing
-
-```Julia
-julia> ] # typing the ] key with go into package REPL mode
-pkg> add ClimaLand
-```
-
-You are now ready to use `ClimaLand.jl`. To get started, we recommend reading the [documentation](https://clima.github.io/ClimaLand.jl/dev/).
+You are now ready to use `ClimaLand.jl`. To get started, we recommend reading the [documentation](https://clima.github.io/ClimaLand.jl/stable/).
 
 ## Models
```

Comment:
I believe @AlexisRenchon wants to add a few lines of code here showing how easy it is to setup and run a simulation! If so, that would be nice to include - either with him taking over this PR or in a followup PR.

(the changes you made look good, so I approved them)

Reply:
I will make a follow up PR!

Reply:
I have a PR open to set up and run a simple soil simulation in `getting_started.md` - maybe we can just point to that? #1267